lib/sw-precache.js

```diff
@@ -183,7 +183,7 @@ function generate(params, callback) {
     if (params.runtimeCaching) {
       var pathToSWToolbox = require.resolve('sw-toolbox/sw-toolbox.js');
       swToolboxCode = fs.readFileSync(pathToSWToolbox, 'utf8')
-        .replace('//# sourceMappingURL=sw-toolbox.map.json', '');
+        .replace('//# sourceMappingURL=./build/sw-toolbox.map.json', '');
```

Comment:
This is at the root on v3.2.1 and looking v3.2.0  the map wasn't added the final bundle